### PR TITLE
fix(app, labware-library): add new NEST reservoir image

### PR DESCRIPTION
**Pull request by ncdiehl11**
# Overview
fix [RAUT-619](https://opentrons.atlassian.net/browse/RAUT-619)

We currently don't have an image associated with our new NEST 1 well reservoir 290ml. Here, I add the image file to relevant images folder within app and labware-library, and also to the corresponding labware images requirements within each.

# Test Plan
- Ensure reservoir image renders in the App and Labware Library

# Changelog
- added image files (.jpg) to App and Labware Library
- added new labware locations to labware images 

# Review requests
General. This is a relatively simple PR, but it is my first contribution to this repo, so any feedback is appreciated.

# Risk assessment
low

[RAUT-619]: https://opentrons.atlassian.net/browse/RAUT-619?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jerader — APPROVED** [review]
woo! lgtm! 🦖

File: app/src/organisms/LabwareDetails/images/nest_1_reservoir_290ml.jpg
Comment (koji) [inline app/src/organisms/LabwareDetails/images/nest_1_reservoir_290ml.jpg:1]: I optimized the image. Could you use this?
Basically most images we use are less 2MB. Highly recommend you to optimize an image/images before adding it/them to the codebase.

1.3mb
nest_1_reservoir_290ml-min


https://squoosh.app/
https://github.com/GoogleChromeLabs/squoosh
https://tinypng.com/
https://tinypng.com/developers

File: labware-library/src/images/nest_1_reservoir_290ml.jpg
Comment (koji) [inline labware-library/src/images/nest_1_reservoir_290ml.jpg:1]: same as the above

**Review by koji — APPROVED** [review]
Once you replace the image, this PR is good to go.

File: app/src/organisms/LabwareDetails/images/nest_1_reservoir_290ml.jpg
Comment (jerader) [reply app/src/organisms/LabwareDetails/images/nest_1_reservoir_290ml.jpg:1]: ooh good catch @koji!

**Comment by ncdiehl11** [comment]
Thanks for the catch— I used the optimized image that attached above (~1.3MB) in the latest commit.

**Comment by koji** [comment]
The target might be `chore_release-7.0.0`?

**Comment by jerader** [comment]
> The target might be `chore_release-7.0.0`?

Yeah, i think you need to target `chore_release-7.0.0` since you are touching the app directory. But i don't know how LL gets deployed -- cc @shlokamin

**Comment by koji** [comment]
@ncdiehl11 
I changed the base from edge to 7.0.0.
https://opentrons.atlassian.net/browse/RAUT-619?focusedCommentId=86024

Now you can merge.

**Pull request merged**